### PR TITLE
2605 - Fix white space on the right side of workspace on mobile view

### PR DESCRIPTION
**Pull request by ericangeles**
**Explain the _details_ for making this change. What existing problem does the pull request solve?**

On mobile, I truncated the title text from `481px (min)` to  `968px (max)` with a text width of `150px` and `250px` from `961px (min)`. So even the title has a very long text, it will be on ellipsis state regardless of the size of the browser.

**Related github/jira issue (required)**:
Closes https://github.com/infor-design/enterprise/issues/2605

**Steps necessary to review your pull request (required)**:
- Pull this branch and run the app
- Navigate to http://localhost:4000/components/contextualactionpanel/example-workspaces.html
- Resize the browser to mobile or use mobile device
- The title text should be truncated and white space in workspace should not be seen.

**Included in this Pull Request**:
- [x] An e2e or functional test for the bug or feature.
- [x] A note to the change log.

### CODE REVIEW

File: docs/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # What's New with Enterprise
 
+## v4.22.0 Fixes
```
Comment (tmcconechy) [inline docs/CHANGELOG.md]: This will make it in 4.21 so you can put this in that section

File: src/components/toolbar-flex/_toolbar-flex.scss [hunk 2]
```diff
@@ -12,6 +12,7 @@
   align-items: center;
   display: flex;
   flex-flow: row nowrap;
+  height: 100%;
```
Comment (tmcconechy) [inline src/components/toolbar-flex/_toolbar-flex.scss]: We removed this due to a bug. So we cannot add a height here. Is there some reason you need this? If for the CAP perhaps can put an explicit height on the toolbar isolated under the contextual-action-panel fx..Add it to this rule instead:

```
.modal.contextual-action-panel .flex-toolbar {
  margin: 0 auto;
  padding: 0 5px;
  height: inherit;
}
```

**Comment by EdwardCoyle** [comment]
@ericangeles I just opened #2624 with a fix for the vertical centering of the Flex Toolbar in CAP.  I'd say to just remove the `height: 100%` in this PR and leave the rest.  When we merge `4.20.x` back into `master` on release, the fix will be present there.

**Comment by ericangeles** [comment]
All good! Thanks @tmcconechy and @EdwardCoyle 🍌

**Pull request merged**